src/legacy/ui/public/vis/editors/default/agg_params.html

```diff
@@ -33,4 +33,16 @@
   </p>
 </div>
 
+<vis-editor-agg-select
+  agg="agg"
+  label="label"
```

Comment:
I would suggest we're passing in a boolean flag `isSubAggregation` instead and keep the translation inside the React component. I think that way we're having the i18n calls in the place they are actually shown, and not in some other controller.